mm2src/coins/lp_coins.rs

```diff
@@ -1461,6 +1484,17 @@ pub enum WithdrawError {
         available: BigDecimal,
         required: BigDecimal,
     },
+    #[display(
+        fmt = "Not enough {} to afford fee. Available {}, required at least {}",
+        coin,
+        available,
+        required
+    )]
+    NotSufficientBalanceForFee {
```

Comment:
The main reason why we have such error is because fees are paid in platform coin. For example if you have 1 IBC/USDC and you want to transfer that, you will pay the fee with IRIS. To avoid confusion, I created new error type in case your balance is enough for the transfer you doing but you don't have enough balance in your platform coin to pay the fee.

Reply:
This makes sense. But we probably should rename it to `NotSufficientPlatformBalanceForFee`. Then I think we need to use this variant in ETH, Lightning, SPL and SLP coins as well.

Reply:
Yeah this seems better, thanks

Reply:
https://github.com/KomodoPlatform/atomicDEX-API/issues/1505